src/control/lib/daos/system_prop_test.go

```diff
@@ -83,6 +91,50 @@ func TestDaos_StringPropVal(t *testing.T) {
 	}
 }
 
+func TestDaos_IntPropVal(t *testing.T) {
+	choices := []int64{1, 2, -3}
+	pv := NewIntPropVal(0, choices...)
+
+	if pv == nil {
+		t.Fatal("expected non-nil NumberPropVal")
+	}
+
+	intChoices := make([]string, len(choices))
+	for i, c := range choices {
+		intChoices[i] = strconv.FormatInt(c, 10)
+	}
+	if diff := cmp.Diff(intChoices, pv.Choices()); diff != "" {
+		t.Fatalf("unexpected choices: (-want,+got)\n%s", diff)
+	}
+
+	if pv.String() != "0" {
```

Comment:
Any reason not to make these a table-driven test?

Reply:
Each set of type tests is just slightly different enough that it seemed like more work to contort them into sets of table tests.